Comp2025/src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -235,6 +240,13 @@ public void periodic( )
     m_leftHeightPub.set(m_leftHeight);
     m_rightHeightPub.set(m_rightHeight);
     m_targetHeightPub.set(m_targetHeight);
+
+    boolean elevatorFullDown = m_elevatorDown.get( );
+
+    if (elevatorFullDown)
```

Comment:
Since we're leaving the original calibration stuff alone (and just now scheduling it), make a new variable called m_homed (meaning the elevator has been sent to the home position at least once for calibration). Use m_homed to determine whether to check for the limit switch, and if the limit is closed while disabled (magnet near the sensor), then set the motor position to zero.